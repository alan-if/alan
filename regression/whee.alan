The kitchen Isa location
  Exit s,e,w,n To hallway.
End The kitchen.

The hallway Isa location
  Exit s,e,w,n To kitchen.
End The hallway.

When Count Isa actor, At kitchen = 1
  Then Schedule wee At hero After 0.

Event wee
  "Whee!"
End Event.

Start At kitchen.
