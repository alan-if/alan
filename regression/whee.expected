########## whee ##########

        No warnings or errors detected.



Kitchen
Whee!

> e
Hallway

> w
Kitchen
Whee!

> 

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
